acir/src/native_types/witness.rs

```diff
@@ -27,23 +27,75 @@ impl Witness {
     pub const fn can_defer_constraint(&self) -> bool {
         true
     }
+}
+
+impl From<u32> for Witness {
+    fn from(value: u32) -> Self {
+        Self(value)
+    }
+}
+
+/// A map from the witnesses in a constraint system to the field element values
+#[derive(Clone, Debug, PartialEq, Eq, Hash, PartialOrd, Ord, Default, Serialize, Deserialize)]
+pub struct WitnessMap(BTreeMap<Witness, acir_field::FieldElement>);
```

Comment:
```suggestion
pub struct WitnessMap(BTreeMap<Witness, FieldElement>);
```

Can we use the non-qualified form here?

Reply:
Was just following what the rest of the module did, but I can hoist it.